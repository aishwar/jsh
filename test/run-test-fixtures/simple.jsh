
# Comment should do nothing

## Simple command
echo Hello

## Simple log message
echo $lang

## Shell evaluated value
echo $nodejs.version

## Deep nested value
echo $nodejs.creator.name
