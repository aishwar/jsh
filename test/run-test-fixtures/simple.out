[LOG]  Simple command
[OUT]  Hello

[LOG]  Simple log message
[OUT]  js

[LOG]  Shell evaluated value
[OUT]  v10

[LOG]  Deep nested value
[OUT]  Ryan Dahl

Script executed
